bin/cheribsdtest/cheribsdtest.c

```diff
@@ -756,6 +756,12 @@ main(int argc, char *argv[])
 			err(EX_OSERR, "setrlimit");
 	}
 
+	/*
+	 * Hook to save the register state in main(), which is the
+	 * intended place to verify register initialization.
+	 */
+	cheribsdtest_snapshot_initregs(cheri_getpcc());
```

Comment:
I don't know that we still need this commit now?  That is, I think the initregs_pcc test still works fine even when reading the current pcc rather than the pcc from main?

Reply:
This is redundant now. Dropped.